include/RAJA/policy/cuda/forall.hpp

```diff
@@ -379,9 +379,9 @@ template<typename EXEC_POL,
                               (IterationGetter::block_size > 0),
                           size_t> BlockSize = IterationGetter::block_size>
 __launch_bounds__(BlockSize, BlocksPerSM) __global__
-    void forall_cuda_kernel(LOOP_BODY loop_body,
-                            const Iterator idx,
-                            IndexType length)
+    void forall_cuda_kernel(const RAJA_CUDA_GRID_CONSTANT LOOP_BODY loop_body,
```

Comment:
So the `__grid_constant__` annotation does not imply `const`? Do you see a technical reason for why it does not? That seems counterintuitive to me.

Reply:
Understood. I looked at the NVIDIA docs. There must be some reason for it.